playwright/components/anchor-navigation/index.ts

```diff
@@ -1,4 +1,4 @@
-import { Page } from "playwright-core";
+import { Page } from "@playwright/test";
```

Comment:
**note**: for future Playwright migrations, we should import any needed TypeScript types from this package as opposed to `playwright-core`, based from recommendation from the Playwright team.